EventFilter/L1TRawToDigi/python/gtStage2Digis_cfi.py

```diff
@@ -3,13 +3,5 @@
 gtStage2Digis = cms.EDProducer(
     "L1TRawToDigi",
     Setup           = cms.string("stage2::GTSetup"),
-    InputLabel      = cms.InputTag("rawDataCollector"),
     FedIds          = cms.vint32( 1404 ),
-    FWId            = cms.uint32(2),
-    lenSlinkHeader  = cms.untracked.int32(8),
-    lenSlinkTrailer = cms.untracked.int32(8),
-    lenAMCHeader    = cms.untracked.int32(8),
-    lenAMCTrailer   = cms.untracked.int32(0),
-    lenAMC13Header  = cms.untracked.int32(8),
-    lenAMC13Trailer = cms.untracked.int32(8)
 )
```

Comment:
Again, why are all parameters removed? If you do have a fillDesciption, then you could remove the explict py files in ../python, (unless you want to have several vaiants)